unit10/unit10.js

```diff
@@ -0,0 +1,145 @@
+//Bai : 1  in một bảng của một số bất kì với các số từ 1 đến 10 và hiển thị ra kết quả 
+const tableMultiplication = (number) => {
+  if(number >= 1 && number <= 10) {
+    console.log('In bảng nhân:');
+    for(let i = 1 ; i <= 10 ; i ++){
+      console.log(number + ' x ' + i + ' = ' + number * i);
+    }
+  }else {
+    console.log('mời bạn nhập lại môt số trong khoảng từ 1 đến 10');
+  }
+}
+
+// Bài 2 : hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 in các số chẵn từ 1 đến n 
+const InEvenNumber = (number) => {
+  if(number >= 1 && number <= 30) {
+    for(let i = 1 ; i <= number ; i ++){
+      if(i % 2 == 0 ){
+        console.log(i);
+      }
+    }
+  }else {
+    console.log('mời bạn nhập lại môt số trong khoảng từ 1 đến 30');
+  }
+}
```

Comment:
Tên hàm: `InEvenNumber` 2 lỗi, 1: (không được viết hoa chữ cái đầu), 2: không phải động từ
Dấu cách line 15: `if(number` -> `if (number`
Dấu cách line 16: `for(let` -> `for (let`
Dấu cách line 16: `for(let i = 1 ;` -> `for(let i = 1;`
Dấu cách line 16: `; i <= number ; i` -> `; i <= number; i`
Dấu cách line 16: `i ++)` -> `i++)`
Dấu cách line 16: `){` -> `) {`
Line 17: `if(i % 2 == 0 ){` -> `if (i % 2 === 0) {` (3 lỗi về dấu cách, lỗi `==` phải dùng `===`)
Line 21: `}else {` -> `} else {`
---> 12 lỗi

```suggestion
const InEvenNumber = number => {
  if (number < 1 || number > 30) return;
  for (let i = 1; i <= number; i ++) {
    if (i % 2 == 0 ) {
      console.log(i);
    }
  }
}
```